share/spice/near_earth_objects/near_earth_objects.js

```diff
@@ -0,0 +1,56 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_near_earth_objects = function(api_result) {
+
+        if (!api_result || !api_result.near_earth_objects || api_result.near_earth_objects.length === 0 || api_result.element_count === 0) {
+            return Spice.failed('near_earth_objects');
+        }
+        
+        // Put all NEOs into a single array
+        var neo_list = [];
+        for (var obj in api_result.near_earth_objects) {
+            if (api_result.near_earth_objects.hasOwnProperty(obj)) {
+                neo_list = neo_list.concat(api_result.near_earth_objects[obj]);
+            }
+        }
+        
+        // Render the response
+        var result_text = (neo_list.length === 1) ? ' near earth object (asteroid)' : ' near earth objects (asteroids)';
+        Spice.add({
+            id: "near_earth_objects",
+            name: "Answer",
+            data: neo_list,
+            
+            meta: {
+                sourceName: 'NASA',
+                sourceUrl: 'http://neo.jpl.nasa.gov/',
+                primaryText: 'Showing ' + neo_list.length + result_text
```

Comment:
The Wikipedia fat-head is triggering on this too, we can set `signal: 'high'` to override (hopefully).